### PR TITLE
RTM and REST error details

### PR DESCRIPTION
**Pull request by skynet67**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/webview/events/ApiErrorDetails.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class ApiErrorDetails {
+    private int code;
+    private String detailedMessage;
+    private String fullMessage;
+
+    public int getCode() {
+        return code;
+    }
+
+    public String getDetailedMessage() {
```
Comment (jeremiahthomasharris) [inline fitpay/src/main/java/com/fitpay/android/webview/events/ApiErrorDetails.java]: Lets add some documentation here about detailed message potentially being null and the code being the http status code. Also include at the top level that it is only used for RTM messages

Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/webview/events/ApiErrorDetails.java]: done

**Comment by skynet67** [comment]
No, but you could subscribe to it. 
https://github.com/fitpay/fitpay-android-sdk/blob/9f13d09053b818e3750dd2aa8ecd6a3407d30aa2/fitpay/src/main/java/com/fitpay/android/webview/impl/parser/RtmParserV5.java#L48-L49

**Comment by jeremiahthomasharris** [comment]
@skynet67 I think we want to log it as well.
Also lets revert the version bump until we are ready to cut a release.

**Comment by skynet67** [comment]
@jeremiahthomasharris without increasing the version it will not be possible to test it. As an option we could name it like 0.6.1-snapshot

**Pull request merged**